services/authoring/src/routes/hooks.ts

```diff
@@ -49,7 +49,7 @@ export class HooksController {
     hook: Hook,
   ): Promise<Hook> {
     const hooksRepository = this.hooksRepositoryFactory.createRepository();
-    hook = { keyPath: hook.keyPath, type: hook.type, url: hook.url };
+    hook = { keyPath: hook.keyPath, type: hook.type, url: hook.url, tags: hook.tags, format: hook.format };
```

Comment:
Probably should use ... as well for code consistency (or alternatively, use exact properties in PUT)

Reply:
done